quality-assurance/test-repositories.md

```diff
@@ -14,6 +14,13 @@ Provides the ability to execute a Full Scan against a web application using the
 
 <https://github.com/UKHO/owasp-zap-scan>
 
+## Playwright Template
```

Comment:
I would suggest putting this (also) in a more prominent position in the page, maybe in the Playwright bit above

Reply:
This page is alphabetical so am reluctant to move it. I have put it in the table of the cross-browser automation page, you think it needs to go elsewhere too?